src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java

```diff
@@ -2,13 +2,16 @@
 
 public class IntLinkedList {
 
+    private static final int EMPTY_VALUE = -1;
+    private static final boolean FULL_STATUS = false;
+
     Node last;
     int i;
 
     public void push(int i) {
-        if (last == null)
+        if (last == null) {
             last = new Node(i);
-        else {
+        } else {
             last.next = new Node(i);
             last.next.prev = last;
```

Comment:
The code does not compile. 
Encapsulation in Node class (Task 11 in "accessory i hermetyzacja") done incompletely?